jwtAuth.go

```diff
@@ -174,10 +194,12 @@ func validateTokenAndExtractClaims(token rawToken, publicKey *rsa.PublicKey) (*T
 }
 
 func extractClaims(parsed *jwt.Token) (*TokenClaims, error) {
-	if !parsed.Valid {
+	if !parsed.Valid && IsVerifyToken() {
```

Comment:
```suggestion
	if IsVerifyToken() && !parsed.Valid {
```
If the env var is set, you never need to evaluate `parsed.Valid` hence slightly speeding up the process.